### PR TITLE
Adds traditional Chinese read me file. And a link to it in README.md

### PR DESCRIPTION
**Pull request by dolphinsue319**

### CODE REVIEW

**Comment by tannernelson** [comment]
This is great! Though I wonder how we should store the files in a different language. Should they all get a `-zh` suffix or should we fork the documentation repo into a separate Chinese one.

**Comment by dolphinsue319** [comment]
I refer to [this project](https://github.com/tiimgreen/github-cheat-sheet). If we use another repo, that is much complex, hard to maintain, and other languages users are hard to find official Vapor documentations in his language.

File: README-Chinese-zh.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+- [Kitura Provider](https://github.com/vapor/kitura-provider): 在 Vapor 中使用 IBM 的 Kitura HTTP Server。
+- [SwiftyBeaver](https://github.com/SwiftyBeaver/SwiftyBeaver-Vapor): 在 Vapor 中使用 SwiftBeaver 的套件。(譯註: 就像強化版的 NSLog() 或 print())
+- [APNS](https://github.com/matthijs2704/vapor-apns): 用來操作 Apple 推播的工具。
+- [JWT](https://github.com/siemensikkema/vapor-jwt): 讓我們可以設定一些規則以取得特定資源的工具。
+- [VaporS3Signer](https://github.com/JustinM1/VaporS3Signer): 用來產生 HTTP request 的 headers 及已經簽證過的 URL，用來 request AWS S3 的 REST API。
+- [Flock](https://github.com/jakeheis/Flock): 自動將 Swift 專案發佈上主機。
+	- [VaporFlock](https://github.com/jakeheis/VaporFlock): 利用 Flock 發佈 Vapor applications。
+
+
+## 作者大大
```
Comment (jhihguan) [inline README-Chinese-zh.md]: 感覺用作者們比較正式一點？

File: README-Chinese-zh.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	- [SQLite Driver](https://github.com/vapor/sqlite-driver): 透迥 Fluent 操作 SQLite 的工具。
+	- [SQLite Provider](https://github.com/vapor/sqlite-provider): 讓 SQLite 可以在 Vapor 上運作的 provider。
+- [PostgreSQL](https://github.com/vapor/postgresql): 用 Swift 操作 PostgreSQL 的工具。
+	- [PostgreSQL Driver](https://github.com/vapor/postgresql-driver): 用 Fluent 操作 PostgreSQL 的套件。
+	- [PostgreSQL Provider](https://github.com/vapor/postgresql-provider): 讓 PostgreSQL 可以運作在 Vapor 上的 provider。
+- [MongoKitten*](https://github.com/OpenKitten/MongoKitten): 用 Swift 寫的 MongoDB driver。
+	- [Mongo Driver](https://github.com/vapor/mongo-driver): Fluent 用的 MongoKitten driver。
+	- [Mongo Provider](https://github.com/vapor/mongo-provider): Vapor 用的 MongoKitten provider.
+	- [MainecoonVapor](https://github.com/OpenKitten/MainecoonVapor): Vapor 的 MongoKitten 物件關聯管理。
+- [Redbird](https://github.com/vapor/redbird): 遵循原原協定的規格實作出來的 Swift Redis client 端。
```
Comment (jhihguan) [inline README-Chinese-zh.md]: just added some opinions about the wording. If those are fine, I can help accept review this!

**Comment by jhihguan** [comment]
just added some opinions about the wording. If those are fine, I can help accept review this!

**Comment by dolphinsue319** [comment]
Fixed, thanks @jhihguan.

**Comment by tannernelson** [comment]
@dolphinsue319 that makes sense. Should we use the same suffixes that they do? Just the language codes? Like `README.zh-cn.md`

**Comment by jhihguan** [comment]
vote for `README.zh-cn.md` and `README.zh-tw.md`
or use Xcode environmen liket `README.zh-hans.md` and `README.zh-hant.md`?
I'm fine with these two

**Comment by dolphinsue319** [comment]
I prefer use `hans` or `hant`, because `cn` or `tw` are area names not language names.

File: README.md [hunk 3]
```diff
@@ -8,6 +8,8 @@ Vapor is the most used web framework for Swift. It provides a beautifully expres
 
 To view the framework's source code and code documentation, visit [Vapor's GitHub](https://github.com/vapor/vapor).
 
+Read this in [正體中文](https://github.com/vapor/documentation/blob/master/README-Chinese-zh.md)
```
Comment (tannernelson) [inline README.md]: Link here should be updated.

**Comment by tannernelson** [comment]
Thanks!

**Pull request merged**